src/translations/en.json

```diff
@@ -151,7 +151,7 @@
       },
       "cover": {
         "position": "Position",
-        "tilt_position": "Tilt position"
+        "tilt_position": "Tilt"
```

Comment:
Why this change?

Reply:
Ah, that snuck in from some other changes for the cover cards I am working on. It sounds weird to me to have "Position" and "Tilt position" since with the first slider we have a real position in terms of "up/down" or "left/right" (e.g. gates). Combining then the words "tilt" and "position" for the second slider feels weird, since now suddenly "position" refers to a different concept (namely the tilt).

Either way, I will remove it from this PR since unrelated topic.